README.md

```diff
@@ -32,18 +43,30 @@ git clone git@github.com:michael-smythe/cvah-train.git
 
 # Quick Start
 This will spin up a new project in your account via terraform, create the custom image via packer, and then deploy an instance of that image with terraform again. It relies on you having already authenticated via the gcloud tool. 
+Take a look at the `quickstart.sh` file and see how to specify the billing account with a Terraform variable in the command `-var "gcp_billing_account_name=My Billing Account FY22"`
 ```bash
-chmod +x ./quickstart
-./quickstart
+chmod +x ./quickstart.sh
+./quickstart.sh
 ```
 
 # New Project
-This spins up a new project under your google account links it to your billing account and enables the appropriate APIs so that you can later build and deploy images and instances. 
+This spins up a new project under your google account links it to your billing account and enables the appropriate APIs so that you can later build and deploy images and instances.
+Specify your billing account with the Terraform variable in the command `-var "gcp_billing_account_name=My Billing Account FY22"`
 ```bash
 cd ./terraform/setup
 terraform init
 terrafrom apply -auto-approve -var 'project_id=<PICK_A_NAME_FOR_THE_PROJECT_ID>'
+
+# Example specifying a billing account:
+# terraform init -var "gcp_billing_account_name=My Billing Account FY22" -var 'project_id=cvah-helk-training-fy22'
```

Comment:
`terraform init` should be sufficient however, this doesn't seem to break anything.